client/cmd/apps.go

```diff
@@ -18,15 +18,13 @@ import (
 )
 
 // AppCreate creates an app.
+
 func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
+	apptype := "buildpack"
 	c, err := client.New()
-	if err != nil {
-		return err
-	}
 
 	fmt.Print("Creating Application... ")
 	quit := progress()
-	app, err := apps.New(c, id)
 
 	quit <- true
 	<-quit
```

Comment:
not familiar with this codebase, but want to make sure. is another goroutine listening on `quit`?